lib/modelManagers/userRoleManager.js

```diff
@@ -0,0 +1,87 @@
+const { Sequelize } = require('sequelize'),
+  { User, Roles, UserRoles } = require('../../database/models');
+
+const { Op } = Sequelize;
+
+/**
+ * @description Model Manager for Article and Admin
+ *
+ * @class UserRoleManager
+ */
+class RoleManager {
+  /**
+   * @description this manager checks admin authentication it check if the user is an admin
+   * @param {*} userId
+   * @static
+   * @returns {*} admin
+   * @memberof RoleManager
+   */
+  static async findUserRole(userId) {
+    const options = {
+      where: {
+        id: {
+          [Op.eq]: userId
+        }
+      },
+      include: [
+        {
+          model: Roles,
+          attributes: ['id', 'role']
+        }
+      ],
+      attributes: ['id', 'firstname', 'lastname', 'email']
```

Comment:
`attributes: ['id', 'firstname', 'lastname', 'email']` Why are we returning this attributes?